examples/simple.js

```diff
@@ -2,10 +2,18 @@ const walmart = require('../src/walmart/index');
 const AmazonClient = require('../src/amazon/AmazonClient');
 const AnalysisClient = require('../src/analysis/AnalysisClient');
 const PairedProductList = require('../src/PairedProductList');
+const RatingsClient = require('../src/ratings/RatingsClient');
+
+//get input parameters specified by user
+let analysisObj = {
+  minROI: process.argv[2],
+  minRatings: process.argv[3],
+  minNumReviews: process.argv[4]
+}
```

Comment:
I think we should still provide default values as well so you can quickly run the program without entering in command line arguments. Whatever parameter values you think will be used the most often. For example,
```
let analysisParameters = {
  minROI: process.argv[2] || 'default',
  minRatings: process.argv[3] || 'default',
  minNumReviews: process.argv[4] || 'default'
}
```

Reply:
Fixed here 905ffb3